addons/mail/static/src/components/activity/activity.xml

```diff
@@ -54,44 +54,46 @@
                                         <t t-esc="activityView.activity.type.displayName"/>
                                     </dd>
-                                </t>
-                                <t t-if="activityView.activity.creator">
-                                    <dt>Created</dt>
-                                    <dd class="o_Activity_detailsCreation">
-                                        <t t-esc="activityView.formattedCreateDatetime"/>
+                                </dl>
+                                <dl t-if="activityView.activity.creator" class="d-md-table-row mb-3">
+                                    <dt class="d-md-table-cell text-md-right m-0 py-md-1 px-md-4">Created</dt>
+                                    <dd class="o_Activity_detailsCreation d-md-table-cell py-md-1 pr-4">
+                                        <t t-esc="activityView.formattedCreateDatetime"/>, by
                                         <img class="o_Activity_detailsUserAvatar o_Activity_detailsCreatorAvatar" t-attf-src="/web/image/res.users/{{ activityView.activity.creator.id }}/avatar_128" t-att-title="activityView.activity.creator.nameOrDisplayName" t-att-alt="activityView.activity.creator.nameOrDisplayName"/>
-                                        <span class="o_Activity_detailsCreator">
+                                        <b class="o_Activity_detailsCreator">
                                             <t t-esc="activityView.activity.creator.nameOrDisplayName"/>
-                                        </span>
+                                        </b>
                                     </dd>
-                                </t>
-                                <t t-if="activityView.activity.assignee">
-                                    <dt>Assigned to</dt>
-                                    <dd class="o_Activity_detailsAssignation">
+                                </dl>
+                                <dl t-if="activityView.activity.assignee" class="d-md-table-row mb-3">
+                                    <dt class="d-md-table-cell text-md-right m-0 py-md-1 px-md-4">Assigned to</dt>
+                                    <dd class="o_Activity_detailsAssignation d-md-table-cell py-md-1 pr-4">
                                         <img class="o_Activity_detailsUserAvatar o_Activity_detailsAssignationUserAvatar" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-title="activityView.activity.assignee.nameOrDisplayName" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
-                                        <t t-esc="activityView.activity.assignee.nameOrDisplayName"/>
+                                        <b t-esc="activityView.activity.assignee.nameOrDisplayName"/>
+                                    </dd>
+                                </dl>
+                                <dl class="d-md-table-row">
```

Comment:
I was hoping that keeping `dt`/`dd` would prevent these big diff indent change, but if you have to introduce one `dl` per line anyway, I think it makes no sense to keep it, and we are better off just using `div` everywhere.